server/backend/database/project_info.go

```diff
@@ -30,6 +30,10 @@ var DefaultProjectID = types.ID("000000000000000000000000")
 // DefaultProjectName is the default project name.
 var DefaultProjectName = "default"
 
+// DefaultClientDeactivateThreshold is the default
+// deactivate threshold of clients in specific project for housekeeping.
+var DefaultClientDeactivateThreshold = 24 * time.Hour
```

Comment:
How about receiving this value from the user when running the server with the CLI?
https://github.com/yorkie-team/yorkie/pull/454/files#diff-8563d4c8ba43b628fe4115162226d50749db653fdcd3dd2e661fb42a4671c3edL227

Reply:
Thank you for your suggestion!

I added `--client-deactivate-threshold` in CLI to set default `Project`'s `ClientDeactivateThreshold` during project creation. Therefore, there are some changes in project creation code to receive CLI's `ClientDeactivateThreshold`.